### PR TITLE
examples: revise StrokeGradient

### PR DESCRIPTION
**Pull request by hermet**
renamed GradientStroke & repositioned visuals.

- Description :

- Issue Tickets (if any) :

- Tests or Samples (if any) :

- Screen Shots (if any) :

### CODE REVIEW

File: src/examples/GradientStroke.cpp [hunk 1]
```diff
@@ -26,23 +26,24 @@ void tvgDrawCmds(tvg::Canvas* canvas)
...
-    shape1->lineTo(50, 100);
     shape1->close();
 
     shape1->stroke(0, 255, 0, 255);
     shape1->stroke(20);
     shape1->stroke(tvg::StrokeJoin::Miter);
     shape1->stroke(tvg::StrokeCap::Butt);
+
     auto fillStroke1 = tvg::LinearGradient::gen();
     fillStroke1->linear(50, 50, 200, 200);
```
Comment (mgrudzinska) [inline src/examples/GradientStroke.cpp:48]: I'd shift the fill, so all 3 colors are visible, but as you wish
fillStroke1->linear(100, 100, 250, 250);

Comment (mgrudzinska) [reply src/examples/GradientStroke.cpp:48]: the same in line 53

Comment (hermet) [reply src/examples/GradientStroke.cpp]: Updated the code, do you satisfied with that? :)

Comment (mgrudzinska) [reply src/examples/GradientStroke.cpp]: that is the result that we get now:
ex_bad


The idea was to check whether a stroke and a fill have the same grad:
ex_ok

You changed the fill differently than I wanted, but it's up to you. In any case, the fill and the stroke's fill should be the same. That's why I wrote that the change also applies to line 53.

**Comment by hermet** [comment]
@mgrudzinska  i hope the last change satisfy with you.

**Comment by mgrudzinska** [comment]
more than happy ;)

**Pull request merged**